src/SqlClient/DataTable.fs

```diff
@@ -7,7 +7,8 @@ open System.Collections.Generic
 open FSharp.Data.SqlClient.Internals
 
 [<Sealed>]
-[<CompilerMessageAttribute("This API supports the FSharp.Data.SqlClient infrastructure and is not intended to be used directly from your code.", 101, IsHidden = true)>]
+//[<CompilerMessageAttribute("This API supports the FSharp.Data.SqlClient infrastructure and is not intended to be used directly from your code.", 101, IsHidden = true)>]
```

Comment:
Is this intentional?

Reply:
Yes: the example project is using the Update function and I don't know the replacement.

Reply:
This:
https://github.com/fsprojects/FSharp.Data.SqlClient/blob/c6e48a47eb1d64c7de7596bae3cb50a548592357/src/SqlClient.Samples/WpfDataBinding/Program.fs#L31